### PR TITLE
Fix some TF slow tests

### PR DESCRIPTION
**Pull request by jplu**
# What does this PR do?

This PR fixes several slow tests related to saved model creation.

### CODE REVIEW

File: tests/test_modeling_tf_common.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def test_saved_model_creation(self):
 
         with tempfile.TemporaryDirectory() as tmpdirname:
             model.save_pretrained(tmpdirname, saved_model=True)
-            saved_model_dir = os.path.join(tmpdirname, "saved_model")
+            saved_model_dir = os.path.join(tmpdirname, "saved_model", "1")
```
Comment (sgugger) [inline tests/test_modeling_tf_common.py:187]: Out of curiosity, why is this change necessary?

**Review by sgugger — APPROVED** [review]
Thanks for fixing!

File: tests/test_modeling_tf_common.py (hunk 1 above)
Comment (jplu) [reply tests/test_modeling_tf_common.py:187]: To be aligned with all the tests that are about saved model. And also because the files are actually created inside `tmpdirname/saved_model/1` and not in `tmpdirname/saved_model`.

**Review by LysandreJik — APPROVED** [review]
LGTM if Patrick approves the Longformer changes

**Review by patrickvonplaten — APPROVED** [review]
Looks good to me!
